poolqc/pooling_qc_sample.py

```diff
@@ -197,45 +203,32 @@ def Summary_table(run_alias, lane_alias):
     return columns, data, style_data_conditional, csv, download
 
 
-def update_sampleindices(run, rows, derived_virtual_selected_rows, colors):
+@app.callback(
+    dep.Output('SampleIndices', 'figure'),
+    [dep.Input('select_a_run', 'value'),
+     dep.Input('lane_select', 'value')])
+def update_sampleindices(run_alias, lane_alias):
+    run = df[(df['Run'] == run_alias) & (df['LaneNumber'] == lane_alias)]
+    run = run[~run['Run'].isna()].drop_duplicates('library')
+    run['Result'] = run['Coding Bases'] / run['Passed Filter Aligned Bases'] * 100
+    run['index'] = run['Index1'].str.cat(
+        run['Index2'].fillna(''), sep=' ')
+    run = run.sort_values('library')
+
+    total_RNA = len(run['library'])
+    Index_Pass = '%s/%s' % (sum(i > 20000 for i in run['SampleNumberReads']), total_RNA)
```

Comment:
This might more accurately be called `samples_passing_clusters` as it's not describing the number of indices which passed.
Please use snake case (lowercase words separated by underscores) for your variable names.